apps/expo-go/ios/Exponent/Kernel/DevSupport/EXPackagerLogHelper.m

```diff
@@ -0,0 +1,164 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+#import "EXPackagerLogHelper.h"
+#import <React/RCTLog.h>
+#import <React/RCTReconnectingWebSocket.h>
+#import <React/RCTBundleURLProvider.h>
+#import <React/RCTConstants.h>
+#import <SocketRocket/SRWebSocket.h>
+
+typedef NS_ENUM(NSInteger, EXLogLevel) {
+  EXLogLevelInfo,
+  EXLogLevelWarning,
+  EXLogLevelError
+};
+
+@interface EXPackagerLogHelper () <SRWebSocketDelegate>
```

Comment:
is it possible to write the new `PackagerLogHelper` in swift rather than objective-c?

Reply:
Since it uses SocketRocket converting to swift fails :(

Reply:
what specific error there? even though SocketRocket doesn't define as a module, we can still add it to the bridging header. we are adding code to expo-go app target, it's fine to use bridging header.

Reply:
Sorry - this works, just me being a bit cross-eyed. I've updated the PR.